lib/ansible/modules/windows/win_domain_group_membership.ps1

```diff
@@ -0,0 +1,155 @@
+#!powershell
+
+# Copyright: (c) 2019, Marius Rieder <marius.rieder@scs.ch>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#Requires -Module Ansible.ModuleUtils.Legacy
+
+try {
+    Import-Module ActiveDirectory
+}
+catch {
+    Fail-Json $result "Failed to import ActiveDirectory PowerShell module. This module should be run on a domain controller, and the ActiveDirectory module must be available."
```

Comment:
I would drop the 'should be run on a domain controller' part, you can run these modules on another host that can communicate with a domain controller and is the reason why we have the `domain_server`, `domain_username`, `domain_password` options to support these scenarios.

Reply:
Changed the error message.